README.md

```diff
@@ -1,7 +1,15 @@
 # Computop Module
 
+[![Build Status](https://travis-ci.org/spryker-eco/computop.svg?branch=master)](https://travis-ci.org/spryker-eco/computop)
+
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/computop/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/computop/?branch=master)
+
 ## Installation
 
 ```
 composer require spryker-eco/computop
-```
\ No newline at end of file
+```
+
+## Documentation
+
+[Documentation](https://academy.spryker.com/developing_with_spryker/industry_partner_integration/partner_integration.html)
```

Comment:
This link leads to nowhere. There is 404 page. Please add link to actual documentation page.

Reply:
Fixed.